desktop-widgets/mainwindow.cpp

```diff
@@ -1571,29 +1573,29 @@ void MainWindow::registerApplicationState(ApplicationState state, Quadrants q)
 	applicationState[(int)state] = q;
 }
 
-void MainWindow::setQuadrantWidget(const Quadrant &q, QSplitter *splitter)
+void MainWindow::setQuadrantWidget(const Quadrant &q, QSplitter &splitter)
```

Comment:
because this always confuses me... why is this better here...?

Reply:
A reference cannot be null and can't be reseated. The reader and the compiler know that there is no point in a null-check.

See: https://godbolt.org/z/1vd55o